interface/weno/adminClass.php

```diff
@@ -0,0 +1,723 @@
+                            `AHFS_descr` varchar(45) NOT NULL,
+                            `NDC` bigint(12) NOT NULL,
+                            `price_per_unit` decimal(5,2) NOT NULL,
+                            `avg_price` decimal(6,2) NOT NULL,
+                            `avg_price_paid` int(6) NOT NULL,
+                            `avg_savings` decimal(6,2) NOT NULL,
+                            `avg_percent` decimal(6,2) NOT NULL,
+                             PRIMARY KEY (`drugid`)
+                             ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=26121" 
+
+    		);
+
+
+    	sqlStatement(
+           "CREATE TABLE IF NOT EXISTS `pharmacies_weno` (
+                        `id` int(5) NOT NULL AUTO_INCREMENT,
+                        `Last_Updated` varchar(21) DEFAULT NULL,
+                        `Store_Name` varchar(35) DEFAULT NULL,
+                        `NCPDP` int(10) DEFAULT NULL,
+                        `Active` varchar(3) DEFAULT NULL,
+                        `NPI` varchar(10) DEFAULT NULL,
+                        `Pharmacy_Phone` varchar(15) DEFAULT NULL,
+                        `Pharmacy_Fax` varchar(15) DEFAULT NULL,
+                        `Address_Line_1` varchar(35) DEFAULT NULL,
+                        `Address_Line_2` varchar(16) DEFAULT NULL,
+                        `City` varchar(16) DEFAULT NULL,
+                        `State` varchar(2) DEFAULT NULL,
+                        `ZipCode` int(5) DEFAULT NULL,
+                        `Retail` varchar(4) DEFAULT NULL,
+                        `Specialty` varchar(5) DEFAULT NULL,
+                        `Long_Term_Care` varchar(5) DEFAULT NULL,
+                        `Mail_Order` varchar(5) DEFAULT NULL,
+                        `Mail_State_Codes` varchar(3) DEFAULT NULL,
+                        `Mail_Address_Line_1` varchar(35) DEFAULT NULL,
+                        `Mail_Address_Line_2` varchar(16) DEFAULT NULL,
+                        `Mail_City` varchar(10) DEFAULT NULL,
+                        `Mail_State` varchar(2) DEFAULT NULL,
+                        `Mail_Zip_Code` int(9) DEFAULT NULL,
+                        `Mail_Phone` int(10) DEFAULT NULL,
+                        `Mail_Fax` int(10) DEFAULT NULL,
+                        `EPCS_Permitted` varchar(5) DEFAULT NULL,
+                        `Accept_NewRx` varchar(4) DEFAULT NULL,
+                        `Accept_RefillResponse` varchar(5) DEFAULT NULL,
+                        `Accept_RxChangeResponse` varchar(5) DEFAULT NULL,
+                        `Accept_Verify` varchar(5) DEFAULT NULL,
+                        `Accept_CancelRx` varchar(5) DEFAULT NULL,
+                        `Accept_RxHistoryRequest` varchar(5) DEFAULT NULL,
+                        `Accept_RxHistoryResponse` varchar(5) DEFAULT NULL,
+                        `Accept_Census` varchar(5) DEFAULT NULL,
+                        `Accept_Resupply` varchar(5) DEFAULT NULL,
```

Comment:
(This comment is in the interface/weno/adminClass.php script)

Better practice to avoid caps in database labels for the future. Using in column labels are ok but frowned upon. table names will break(which you are not doing).